cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -76,6 +76,31 @@ std::string GetMinElementOrDefault(const std::vector<std::string> &elements, con
 }
 
 
+struct Journal {
+    unsigned id_;
```

Comment:
What is this ID?  Either a name change or a comment might be nice.

Reply:
Its the journal id => primary key in the database. I normally don't write this as prefix if it belongs to the main object. That's why i also called the other property "name_" and not "journal_name_". Usually you access this on a variable which is e.g. defined as `Journal journal;` and then you only need to write `journal.id_` instead of `journal.journal_id_` so it means less typing. Only in case of e.g. `zeder_journal_` it is useful to add a prefix, to make it clear that this is something special or external and no default property.